EIPS/eip-7069.md

```diff
@@ -80,22 +78,23 @@ Execution semantics of `EXT*CALL`:
 11. Perform the call with the available gas and configuration.
 12. Push a status code on the stack:
     - `0` if the call was successful.
-    - `1` if the call has reverted (also can be pushed earlier in a light failure scenario).
-    - `2` if the call has failed.
+    - `1` if the call has reverted (also can be pushed earlier in a light failure scenario described in step 10.).
+    - `2` if the call has failed. (In case of general OOG failure or any scenario where all gas passed to the callee is burnt in case of an error)
```

Comment:
```suggestion
    - `2` if the call has failed (in case of general OOG failure or any scenario where all gas passed to the callee is burnt in case of an error).
```